quickstarts/aws/aws-vpn/config.yml

```diff
@@ -0,0 +1,32 @@
+slug: aws-vpn
+description: |-
+  ## What is AWS VPN?
+  Amazon Virtual Private Cloud is a commercial cloud computing service that provides a virtual private cloud, by provisioning a logically isolated section of Amazon Web Services Cloud. Enterprise customers can access the Amazon Elastic Compute Cloud over an IPsec based virtual private network.
+  ### Get started!
+  Start monitoring AWS VPN by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-vpc-monitoring-integration/) to learn more about New Relic monitoring for AWS VPN.
+summary: |-
+  Monitor AWS VPN by connecting AWS to New Relic
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: AWS VPN
+documentation:
+  - name: AWS VPN installation docs
+    description: |
+      Monitor AWS VPN by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-vpc-monitoring-integration/
+keywords:
+  - aws
+  - amazon web services
+  - aws vpn
+  - VPN
+dashboards:
+  - aws-vpn
+dataSourceIds:
+  - amazon-cloudwatch-metric-streams
+alertPolicies:
+  - aws-vpn
```

Comment:
It looks like validation is failing because this alert policy doesn't exist

Reply:
Thanks @sarahkitten, I made the required changes.